README.md

```diff
@@ -26,11 +26,12 @@ Goose is your on-machine developer agent, automating engineering tasks seamlessl
 
 <p align="center">
 <a href="#unique-features-of-goose-compared-to-other-ai-assistants">Unique features</a> 🤖 •
-<a href="#what-block-employees-have-to-say-about-goose"> Block Employees on Goose</a> <img src=docs/assets/logo.png height="15", width="15" alt="Block Emoji"/> •
+<a href="#what-block-employees-have-to-say-about-goose"> Block Employees on Goose</a> <img src=assets/logo.png height="15", width="15" alt="Block Emoji"/> •
```

Comment:
Pro: renders in the docs now because we are using the path `assets/logo.png`
Con: readme is now not pretty